### PR TITLE
Add: 12.0-beta1 announcement

### PR DESCRIPTION
**Pull request by TrueBrain**
Reddit: just the link: https://www.openttd.org/news/2021/08/15/openttd-12-0-beta1.html

Discord:
```
Hi @everyone,

I noticed it was over three months since we made our last release. Unacceptable. So it is time we start our OpenTTD 12 release cycle. Let's start with a beta .. just to make sure we didn't make any big boo-boos.

What is new you ask? Owh, just a way for you to play together really easy with your friends. Forget about port-forwarding .. just .. forget about it. Never again.

Not a multiplayer fan? No worries, we got plenty of other new features for you! Maybe you want to try a game with 255 NewGRFs?

Please report any issues you find with this beta in the usual place. It helps a lot getting the next stable out quickly and as bug-free as possible!

https://www.openttd.org/news/2021/08/15/openttd-12-0-beta1.html
```

Twitter:
```
OpenTTD 12.0-beta1 now available on Steam / our website.
Multiplayer reworked. Needs testing! Please help?
https://www.openttd.org/news/2021/08/15/openttd-12-0-beta1.html
```

tt-forums:
```
Yes. Really. 12.0-beta1. Out now! Check the news for more information!

https://www.openttd.org/news/2021/08/15/openttd-12-0-beta1.html
```

Steam:
Website post with this image:
OpenTTD - News #6

### CODE REVIEW

**Comment by James103** [comment]
Do you know why you are skipping to version 12.0 instead of just incrementing to 1.12.0?

**Comment by LordAro** [comment]
Read the post!

**Comment by TrueBrain** [comment]
Created text for socials. Steam image is not the best work, but I am still hoping someone is annoyed enough they take over that part :P

File: _posts/2021-08-15-openttd-12-0-beta1.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+Sadly, to set expectations, no integration with Steam itself has been added.
+Integrating with Steam as GPL-licenses software has challenges, which we haven't resolved yet.
+But we hope you agree that this new multiplayer experience is the next best thing.
+
+Next to this excellent new feature, a few other new things we introduced are:
+* Brand new "Online Players" window to easier see who is in what company, etc.
+* Maximum number of NewGRFs you can use in one game has been raised to 255.
+* Our official Windows builds are now signed.
+* Define refittability of default vehicles using cargo classes.
```
Comment (andythenorth) [inline _posts/2021-08-15-openttd-12-0-beta1.md:27]: `Default vehicles can now be refitted to multiple cargos when used with industry NewGRFs.`

or

`Default vehicles are now compatible with almost all industry NewGRFs`

File: _posts/2021-08-15-openttd-12-0-beta1.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+Integrating with Steam as GPL-licenses software has challenges, which we haven't resolved yet.
+But we hope you agree that this new multiplayer experience is the next best thing.
+
+Next to this excellent new feature, a few other new things we introduced are:
+* Brand new "Online Players" window to easier see who is in what company, etc.
+* Maximum number of NewGRFs you can use in one game has been raised to 255.
+* Our official Windows builds are now signed.
+* Default vehicles are now compatible with almost all industry NewGRFs.
+* Configurable subsidy duration, up to 5000 years.
+* Press Ctrl to build diagonal rivers in Scenario Editor
```
Comment (andythenorth) [inline _posts/2021-08-15-openttd-12-0-beta1.md:29]: missing `.`

**Review by andythenorth — DISMISSED** [review]

File: _posts/2021-08-15-openttd-12-0-beta1.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+With our release on Steam not even six months ago we got some excellent feedback from the community on what to improve next.
+One of the things that stood out: playing together is not easy for many players.
+
+So, in 12.0 we address this problem.
+Playing together is as easy as one of you starting a server, marking it either "Public" or "Invite-Only", and sharing the newly introduced invite-code with your friends.
+They can join your server, just like that.
+No setup needed on your side, no port-forwarding, nothing.
+
+Sadly, to set expectations, no integration with Steam itself has been added.
+Integrating with Steam as GPL-licenses software has challenges, which we haven't resolved yet.
```
Comment (James103) [inline _posts/2021-08-15-openttd-12-0-beta1.md:20]: The line could instead be:
> Integrating with Steam as GPL-licensed software has challenges, which we haven't resolved yet.

**Review by andythenorth — APPROVED** [review]

**Pull request merged**